frontside/lib/utils/debouncer.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+class Debouncer {
+  Duration delay;
+  Timer? _timer;
+
+  Debouncer({this.delay = const Duration(milliseconds: 500)});
+
+  void debounce(VoidCallback callback) {
+    if (_timer != null && _timer!.isActive) {
+      _timer!.cancel();
+    }
+    _timer = new Timer(delay, callback);
```

Comment:
you don't need and from flutter 2 shouldn't call new keyword

Reply:
728028b